frame/staking/src/pallet/impls.rs

```diff
@@ -1451,3 +1451,103 @@ impl<T: Config> StakingInterface for Pallet<T> {
 		Nominators::<T>::get(who).map(|n| n.targets.into_inner())
 	}
 }
+
+#[cfg(feature = "try-runtime")]
+impl<T: Config> Pallet<T> {
+	pub(crate) fn do_sanity_check(_: BlockNumberFor<T>) -> Result<(), &'static str> {
```

Comment:
I think the renaming to `integrity_check` is still a good idea.

Reply:
But then this name is taken for something else :( 

I will probably rename them in this PR as well.